src/print.rs

```diff
@@ -403,3 +395,66 @@ pub fn format_duration(duration: &Duration) -> String {
         format!("{:?}ms", &milis)
     }
 }
+
+// Return the modules from $all that are not already in the list
```

Comment:
**nitpick:** triple-slashes to allow editors to show the description as part of code completion.

```suggestion
/// Return the modules from $all that are not already in the list
```